properties.schema

```diff
@@ -14,9 +14,10 @@
     },
     "item": {
       "type": "string",
-      "title": "Item",
+      "title": "Item count label",
       "default": "Item {{{itemNumber}}} of {{{totalItems}}}",
       "inputType": "Text",
+      "help": "This is the aria label for each item. Use {{itemNumber}} and {{totalItems}} in your text to tell the user which item they are viewing and how many items there are in total.",
```

Comment:
All working as expected, just missing an extra set of curly braces from the help to match default.

Reply:
Hey @joe-allen-89, I've just added an extra set of curly braces to both schema `help` and `description` text. On reflection, do we even need the third curly for `itemNumber` and `totalItems`?

Reply:
I assumed they'd been added for a reason, but looking back over commits they may have been added accidentally during the schema work, as we're only expecting numbers back double curly braces should be sufficient.

Reply:
Thanks @joe-allen-89, third curly braces removed in [c873481](https://github.com/adaptlearning/adapt-contrib-hotgraphic/pull/292/commits/c873481740ad0263ec130460efd7aa452bb8ebd2) and [a8865f7](https://github.com/adaptlearning/adapt-contrib-hotgraphic/pull/292/commits/a8865f77f916687675516a4d2f4a669beec4c6da).